### PR TITLE
Fixed grammar and help

### PR DESCRIPTION
**Pull request by joel-hamill**
While documenting the updated CLI commands (dcos/dcos-docs#874), I noticed these items that needed fixed. This is a replacement for https://github.com/dcos/dcos-cli/pull/894.

### CODE REVIEW

**Comment by joel-hamill** [comment]
@tamarrow any ideas?

File: cli/dcoscli/data/help/node.txt [hunk 1]
```diff
@@ -1,42 +1,52 @@
...
+    dcos node diagnostics download <bundle> [--location=<location>]
     dcos node list-components [--leader --mesos-id=<mesos-id> --json]
-    dcos node ssh [--option SSHOPT=VAL ...]
+    dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<agent-id>]
+                  [--component=<component-name> --filter=<filter>...]
+    dcos node ssh (--leader | --master | --mesos-id=<mesos-id> | --slave=<agent-id>)
                   [--config-file=<path>]
                   [--user=<user>]
                   [--master-proxy]
+                  [--option SSHOPT=VAL ...]
```
Comment (tamarrow) [inline cli/dcoscli/data/help/node.txt]: @joel-hamill the `[<command>]` option is missing

File: cli/dcoscli/data/help/node.txt [hunk 2]
```diff
@@ -1,42 +1,53 @@
...
+    dcos node diagnostics (--list | --status | --cancel) [--json]
+    dcos node diagnostics create (<nodes>)...
+    dcos node diagnostics delete <bundle>
+    dcos node diagnostics download <bundle> [--location=<location>]
     dcos node list-components [--leader --mesos-id=<mesos-id> --json]
-    dcos node ssh [--option SSHOPT=VAL ...]
+    dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<agent-id>]
+                  [--component=<component-name> --filter=<filter>...]
+    dcos node ssh (--leader | --master | --mesos-id=<mesos-id> | --slave=<agent-id>)
+                  [<command>]
```
Comment (tamarrow) [inline cli/dcoscli/data/help/node.txt]: since this is a positional argument it needs to be last on the list of options

**Comment by joel-hamill** [comment]
@tamarrow do you know what additional fixes are needed?

**Comment by tamarrow** [comment]
None. There's a marathon issue that's causing these test failures. Once the fix is merged to testing/master I'll rerun the tests and merge :)

**Pull request merged**